eclipse-scout-core/src/tree/TreeNode.ts

```diff
@@ -425,6 +418,9 @@ export class TreeNode implements TreeNodeModel, ObjectWithType, FilterElement {
     if ($node.hasClass('parent-of-selected')) {
       cssClass += ' parent-of-selected';
     }
+    if ($node.hasClass('focused')) {
```

Comment:
Ok, but why is this method called _preserveCssClasses? It seems to be used to remove all classes expect a few. But what are they?

Reply:
This is not a new method, the js doc explains it. If you have a better name, we can change it